### PR TITLE
DDF-3691 Reintroduce testCertificateGeneratorService with try loop

### PR DESCRIPTION
**Pull request by ahoffer**
#### What does this PR do?
Adds back in an itest.
#### Who is reviewing it? 
@tbatie  @mcalcote @vinamartin 

#### Choose 2 committers to review/merge the PR. 
@clockard
@figliold
@rzwiefel
@shaundmorris
#### How should this be tested? (List steps with links to updated documentation)
full CI build
#### What are the relevant tickets?
DDF-3691

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java [hunk 1]
```diff
@@ -1250,6 +1255,61 @@ void restoreKeystoreFile() throws IOException {
...
+                Response response =
+                    given()
+                        .auth()
+                        .preemptive()
+                        .basic("admin", "admin")
+                        .when()
+                        .get(certGenPath + "/configureDemoCert/" + commonName);
+                String actualValue =
+                    JsonPath.from(response.getBody().asString()).getString("value");
+                return actualValue.equals(expectedValue);
```
Comment (ahoffer) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java]: Can you spot the bug on line 1288?

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now

**Comment by ahoffer** [comment]
build now
Git failed to fetch PR branch or master branch from SCM

File: distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java [hunk 2]
```diff
@@ -1250,6 +1257,61 @@ void restoreKeystoreFile() throws IOException {
...
+        SECURE_ROOT_AND_PORT
+            + "/admin/jolokia/exec/org.codice.ddf.security.certificate.generator.CertificateGenerator:service=certgenerator";
+    getBackupKeystoreFile();
+    try {
+
+      getServiceManager().startFeature(true, featureName);
+
+      //Test first operation
+      expect("Wait for MBeanServer to register object")
+          .within(30, SECONDS)
```
Comment (peterhuffer) [inline distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java]: ❓ Maybe 1-5 minutes would be more consistent? Or is that too much?

**Comment by ahoffer** [comment]
Now that the OWASP upgrade has been merged, it makes sense to close this and start with a fresh branch; this branch is polluted with suppressions and changes that are now part of the master branch.

**Pull request closed without merging**